app/src/main/java/com/example/drawingactivity/DrawViewModel.kt

```diff
@@ -213,8 +219,18 @@ class DrawViewModel : ViewModel() {
                 isDeleteDialogOpen,
                 isBottomSheetOpen,
                 isColorPickerOpen,
-                previousHSVColor
+                previousHSVColor,
+                toolLayoutOpen
```

Comment:
Always add trailing commas so the git diff doesn't have to have 2 added lines when you add one

Reply:
added a trailing comma, interesting to know